tests/cli.rs

```diff
@@ -32,4 +62,23 @@ fn create_output_dir() {
     ])
     .assert()
     .failure();
+
+    let mut cmd = Command::cargo_bin("solang").unwrap();
+
+    cmd.args([
+        "compile",
+        "examples/flipper.sol",
+        "--target",
+        "solana",
+        "--contract",
+        "flapper,flipper", // not flipper
+        "--output",
+        "tests/create_me",
+    ])
+    .assert()
+    .failure();
+
+    remove_file("tests/create_me/flipper.json").unwrap();
+    remove_file("tests/create_me/flipper.so").unwrap();
```

Comment:
Are we only building the contracts we found? I though we would ignore everything if any of the contracts specified could not be found.

Reply:
Fixed